runelite-client/src/test/java/net/runelite/client/util/TextTest.java

```diff
@@ -42,4 +42,12 @@ public void removeTags()
 		assertEquals("Remove no tags", Text.removeTags("Remove no tags"));
 	}
 
+	@Test
+	public void unescapeTags()
```

Comment:
This should include cases where tags are removed, since that is included as part of the behavior